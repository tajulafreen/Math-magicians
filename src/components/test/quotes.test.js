import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../Quotes';
import '@testing-library/jest-dom/extend-expect';

describe('Quote component', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Quote />);
    expect(container).toMatchSnapshot();
  });
});
