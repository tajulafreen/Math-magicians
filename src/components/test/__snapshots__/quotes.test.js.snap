// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component should match the snapshot 1`] = `
<div>
  <div
    class="quote-Container"
  >
    <h2
      class="quote-Header"
    >
      Facts
    </h2>
    <p
      class="loading"
    >
      Loading for quote...
    </p>
  </div>
</div>
`;
